gui/wxpython/startup/guiutils.py

```diff
@@ -211,6 +211,27 @@ def create_mapset_interactively(guiparent, grassdb, location):
     return mapset
 
 
+def create_location_interactively(guiparent, grassdb):
+    """
+    Location wizard started
+    """
+    from location_wizard.wizard import LocationWizard
+    gWizard_output = None
+    try:
+        gWizard = LocationWizard(parent=guiparent,
+                                 grassdatabase=grassdb)
+        gWizard_output = (gWizard.grassdatabase, gWizard.location,
+                          gWizard.georeffile, gWizard.default_region,
+                          gWizard.user_mapset)
+    except OSError as err:
```

Comment:
Does LocationWizard raise OSError? Otherwise, same questions as for OnDownloadLocation() in tree.py.

Reply:
Same as above.